graph/src/schema/api.rs

```diff
@@ -71,6 +73,220 @@ impl TryFrom<&r::Value> for ErrorPolicy {
     }
 }
 
+#[derive(Debug)]
+pub struct ApiSchema {
```

Comment:
Comment would be great

Reply:
This is code that I moved here from somewhere else as-is; I'll add a comment